specification/network/resource-manager/Microsoft.Network/2015-05-01-preview/network.json

```diff
@@ -6197,6 +6197,10 @@
             "$ref": "#/definitions/Usage"
           },
           "description": "Gets or sets the list Network Resource Usages."
+        },
+        "nextLink": {
```

Comment:
Example of this propery being added exactly like this in a later API Version:

https://github.com/Azure/azure-rest-api-specs/blob/7c6c49a6d8507db1b70909b6b380d7f3bd4168cc/specification/network/resource-manager/Microsoft.Network/2017-06-01/usage.json#L140